apps/explorer/tsconfig.json

```diff
@@ -17,7 +17,8 @@
         "jsx": "react-jsx",
         "baseUrl": ".",
         "paths": {
-            "~/*": ["./src/*"]
+            "~/*": ["./src/*"],
+            "@mysten/core": ["../core/src/"],
```

Comment:
We could probably avoid this and just set the `package.json` `main` field to `src/index.ts`. I think this is what we're doing for icons as well.

Reply:
fixed, thanks!